### PR TITLE
Report builder tiles

### PR DESCRIPTION
**Pull request by nickpell**
screen shot 2015-07-09 at 2 28 21 pm

@biyeun @amsagoff cc: @orangejenny

### CODE REVIEW

**Comment by nickpell** [comment]
@biyeun Help text isn't working - does this need angular?

File: corehq/apps/userreports/templates/userreports/builder_report_type_select.html [hunk 1]
```diff
@@ -37,38 +42,37 @@
...
-                        {% trans 'Data Table' %}
-                        <span class="hq-help-template" data-content="{% trans "A table of aggregated data from form submissions or case properties. You choose the columns and rows." %}"></span>
-                    </button>
-                </a>
-            </div>
+            {% for tile in tiles %}
+            <tile-{{ tile.ng_directive }}
+                    slug="{{ tile.slug }}"
+                    title="{{ tile.title }}">
+                <div class="col-md-3">
```
Comment (biyeun) [inline corehq/apps/userreports/templates/userreports/builder_report_type_select.html]: get rid of this if you're already doing span3

**Comment by biyeun** [comment]
@nickpell yes, you should include angular (see what norman did in the dashboard for app builder again)

File: corehq/apps/userreports/templates/userreports/builder_report_type_select.html [hunk 2]
```diff
@@ -37,38 +42,37 @@
...
+                        <div class="panel-body">
+                            <a href="{{ tile.url }}"
+                               class="dashboard-link"
+                               track-analytics=""
+                               process-external-link=""
+                               popover-title="{{ tile.title }}"
+                               popover="{{ tile.helpText }}"
+                               popover-trigger="mouseenter"
+                               popover-placement="top"
+                               title="">
```
Comment (orangejenny) [inline corehq/apps/userreports/templates/userreports/builder_report_type_select.html]: any particular reason to specify track-analytics, process-external-link, and title as blank? just curious.

Comment (nickpell) [inline corehq/apps/userreports/templates/userreports/builder_report_type_select.html]: nope, just got transferred during copy/paste

**Comment by orangejenny** [comment]
Any idea what the timeframe is for this to be migrated to bootstrap 3?

Looks fine to me, will leave for @biyeun to merge

**Comment by biyeun** [comment]
@nickpell will merge after you remove `col-md-3` div

File: corehq/apps/userreports/templates/userreports/builder_report_type_select.html (hunk 1 above)
Comment (nickpell) [inline corehq/apps/userreports/templates/userreports/builder_report_type_select.html]: when I get rid of ```col-md-3``` it turns into

screen shot 2015-07-09 at 6 09 25 pm

Comment (biyeun) [inline corehq/apps/userreports/templates/userreports/builder_report_type_select.html]: sorry, make it into col-sm-2 (meaning for small screens, make it two columns wide)

Comment (nickpell) [inline corehq/apps/userreports/templates/userreports/builder_report_type_select.html]: This is what happens when I try col-sm-2:
screen shot 2015-07-10 at 10 37 35 am

Comment (nickpell) [inline corehq/apps/userreports/templates/userreports/builder_report_type_select.html]: I think the problem is that ```.dashboard-icon-medium``` takes precedence over the resizing, but not sure how to fix

Comment (biyeun) [inline corehq/apps/userreports/templates/userreports/builder_report_type_select.html]: you can add a new class that's dashboard-icon-small. Are you wrapping the columns directly in a class="row" div?

**Comment by nickpell** [comment]
looking better now:

screen shot 2015-07-10 at 4 03 32 pm

**Comment by nickpell** [comment]
:white_check_mark:

**Comment by biyeun** [comment]
fix merge issues, but other than that :+1:

**Comment by nickpell** [comment]
merge conflict fixed

**Comment by NoahCarnahan** [comment]
Looks great @nickpell, although I think the help text isn't working?

**Comment by nickpell** [comment]
yeah, I talked to @amsagoff about that.  it requires figuring out angular so it's going to be resourced explicitly

**Comment by NoahCarnahan** [comment]
cool, good to know.

**Pull request merged**